roles/generate-jenkins/templates/Jenkinsfile.j2

```diff
@@ -1201,6 +1201,71 @@ pipeline {
+      when {
+        branch "{{ ls_branch }}"
+        environment name: 'CHANGE_ID', value: ''
+        environment name: 'EXIT_STATUS', value: ''
+      }
+      steps {
+        echo "Setting up protection for release branch {{ ls_branch }}"
+        sh '''#! /bin/bash
+          curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST https://api.github.com/repos/${LS_USER}/${LS_REPO}/branches/{{ ls_branch }}/protection \
+          -d $(jq -c .  << EOF
+            {
+              "required_status_checks": null,
+              "enforce_admins": true,
+              "required_pull_request_reviews": {
+                "dismiss_stale_reviews": false,
+                "require_code_owner_reviews": false,
+                "require_last_push_approval": false,
+                "required_approving_review_count": 1
+              },
+              "restrictions": null,
+              "required_linear_history": false,
+              "allow_force_pushes": false,
+              "allow_deletions": false,
+              "block_creations": false,
+              "required_conversation_resolution": true,
+              "lock_branch": false,
+              "allow_fork_syncing": false,
+              "required_signatures": false
+            }
+EOF
+          ) '''
+      }
+    }
+    // Add stale exempt labels to the repository
+    stage('Github-Stale-Exempt-Labels') {
+      when {
+        branch "{{ ls_branch }}"
+        environment name: 'CHANGE_ID', value: ''
+        environment name: 'EXIT_STATUS', value: ''
+      }
+      steps {
+        echo "Setting up stale exempt labels for ${LS_USER}/${LS_REPO}"
+        sh '''#! /bin/bash
+          # awaiting-approval
+          if curl -H "Authorization: token ${GITHUB_TOKEN}" https://api.github.com/repos/${LS_USER}/${LS_REPO}/labels/awaiting-approval | grep -q 'awaiting-approval'; then
+            curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST https://api.github.com/repos/${LS_USER}/${LS_REPO}/labels/awaiting-approval \
+              -d '{"description":"Stale exempt"}'
+          else
+            curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST https://api.github.com/repos/${LS_USER}/${LS_REPO}/labels \
+            -d '{"name":"awaiting-approval","description":"Stale exempt"}'
+          fi
+          
+          # work-in-progress
+          if curl -H "Authorization: token ${GITHUB_TOKEN}" https://api.github.com/repos/${LS_USER}/${LS_REPO}/labels/work-in-progress | grep -q 'work-in-progress'; then
+            curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST https://api.github.com/repos/${LS_USER}/${LS_REPO}/labels/work-in-progress \
+              -d '{"description":"Stale exempt"}'
+          else
+            curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST https://api.github.com/repos/${LS_USER}/${LS_REPO}/labels \
+            -d '{"name":"work-in-progress","description":"Stale exempt"}'
+          fi
+        '''
+      }
+    }
```

Comment:
This could also live in the "issues-cron" action. downside is that it would take a week to propagate to new repos

Reply:
Resolved in https://github.com/linuxserver/docker-jenkins-builder/pull/208/commits/9ef71cec212da6f9d0ac2451087cbc82e58be81e / https://github.com/linuxserver/github-workflows/pull/36